tools/pythonpkg/setup.py

```diff
@@ -160,10 +160,18 @@ def __str__(self):
 extra_files = []
 header_files = []
 
+def list_source_files(directory):
+    sources = []
+    dir_len = len(directory) + len(os.sep)
+    for (subdir, dirs, files) in os.walk(directory):
+        subdir = subdir[dir_len::]
```

Comment:
Please use relpath instead of this

Or use glob to replace the entire method maybe?

Reply:
My main concern is future legibility than anything